web/models.py

```
@@ -1752,6 +1752,65 @@ def display_name(self):
+        self.full_clean()  # Call full_clean to enforce validation
+        if not self.slug:
+            self.slug = slugify(self.title)
+            if Meetup.objects.filter(slug=self.slug).exists():
+                suffix = uuid.uuid4().hex[:6]
+                self.slug = f"{self.slug}-{suffix}"
+        super().save(*args, **kwargs)
+
+    def __str__(self):
+        return self.title
+
+    def can_edit(self, user):
```

Ruff (0.8.2)

Missing return type annotation for public function `can_edit`

(ANN201)